app/server/lib/Authorizer.ts

```diff
@@ -600,6 +600,11 @@ export function assertAccess(
     throw new ErrorWithCode("AUTH_NO_VIEW", "Document is deleted", {status: 404});
   }
 
+  // Disabled docs in the process of being removed (deleted) are allowed,
```

Comment:
What is being allowed, specifically?

Reply:
Deleting or undeleting a doc. I've rephrased.